### ISSUE TITLE
Support for tablesorter?

### ISSUE THREAD

**Issue by fschroiff** [zurb/responsive-tables#4]
Your plugin splits the table into two tables, tablesorter still works on the second table, not on the first though. Any ideas on how to solve this?
Tablesorter plugin: http://tablesorter.com/docs/

**Comment by thomas-gordon** [comment]
yeah I reckon that might indeed be possible. I'll hit it up soon.

**Comment by bob-a** [comment]
FWIW, it works great with http://datatables.net for sorting.

**Comment by stl99** [comment]
Hi there,

Any news about this? I recently integrated Zurb responsive tables and realized too late that it doesn't work with the tablesorter plugin which is essential for me..

Cheers,
Thomas
